OWNERS

```diff
@@ -5,15 +5,15 @@ approvers:
   - DjangoPeng
   - gaocegege
   - jlewi
-  - llunn
+  - lluunn
   - ScorpioCPH
 reviewers:
   - ankushagarwal
   - DjangoPeng
   - gaocegege
   - Jimexist
   - jlewi
-  - llunn
+  - lluunn
```

Comment:
Can you also remove everyone who is in the approvers list from the reviewer list?
https://github.com/kubeflow/community/blob/master/contributing.md look at the last bullet point about good examples.